### ISSUE TITLE
Protein vectors in protein-interaction-network

### ISSUE THREAD

**Issue by datduong** [bio-ontology-research-group/deepgo#24]
The paper cites the protein vectors in Alshahrani,M. et al. (2017). Did you use exactly the same vectors by Alshahrani,M. et al. (2017)? In this paper Alshahrani,M. et al. (2017), there 6 other databases which were used to encode proteins into vectors. So just to confirm, are the protein vectors contain only data from interaction-network, or from other additional sources? Thanks.

**Comment by coolmaksat** [comment]
No, we have built a new graph from PPI and orthology links from EGGNOG

**Comment by datduong** [comment]
Awesome. Thanks. One last question, is the method applied to your own PPI data exactly the same as Alshahrani,M. et al. (2017)?
